numpy/lib/function_base.py

```diff
@@ -1487,37 +1487,44 @@ def _unwrap_dispatcher(p, discont=None, axis=None):
 
 
 @array_function_dispatch(_unwrap_dispatcher)
-def unwrap(p, discont=pi, axis=-1):
-    """
-    Unwrap by changing deltas between values to 2*pi complement.
+# def _unwrap_dispatcher(p, discont=None, axis=None, *, min_val=None, max_val=None):
+#     return (p,)
 
-    Unwrap radian phase `p` by changing absolute jumps greater than
-    `discont` to their 2*pi complement along the given axis.
 
+# @array_function_dispatch(_unwrap_dispatcher)
+def unwrap(p, interval_size= 2*pi, discont=None, axis=-1):
```

Comment:
Changing the order of arguments breaks compatibility - I'd recommend you make this
```suggestion
def unwrap(p, discont=None, axis=-1, *, interval_size=2*pi):
```